src/robot_arm.cpp

```diff
@@ -1,37 +1,79 @@
 /**
  * @file
  * @brief     Robot Arm class
- * @author    Jeroen van Hattem and Jeffrey de Waal
- * @license   MIT
+ * @author    Jeroen van Hattem, Jeffrey de Waal, Wiebe van Breukelen, Sam Zandee
+ * @license   See LICENSE
  */
 #include "robot_arm.hpp"
 
 namespace RobotArm {
 
-RobotArm::RobotArm() : conn(115200, UARTController::ONE) {
+RobotArm::RobotArm() : uartConn(115200, UARTController::ONE) {
 }
 
 void RobotArm::sendGCodeToArm(const char *command) {
-    conn << command;
+    uartConn << command;
 }
 
-void RobotArm::move(const Coordinate3D coordinates, int speed) {
-    speed = speed;
+void RobotArm::move(Coordinate3D coordinates, unsigned int speed) {
+    this->speed = speed;
```

Comment:
Try to avoid using a this pointer when possible.
Instead use something like speed = newSpeed like in the exeuteAction function.